tests/test_0293-cuda-allocation-tests.cpp

```diff
@@ -0,0 +1,95 @@
+// BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/master/LICENSE
+
+#include <memory>
+
+#include <awkward/Index.h>
+#include "awkward/kernel.h"
+#include "awkward/array/NumpyArray.h"
+
+namespace ak = awkward;
+
+// The extra steps needed to enable CUDA kernels on pure C++
+class StartupLibraryPathCallback : public kernel::LibraryPathCallback {
+public:
+    StartupLibraryPathCallback() = default;
+
+    const std::string library_path() const override {
+      std::string library_path = ("/path/to/python-pip/awkward1_cuda_kernels/libawkward-cuda-kernels.so");
```

Comment:
This is a test I'll need to modify manually to use, right?

Reply:
Yeah, unless I have the python module ready, unfortunately this is the way to test `awkward1-cuda-kernels`.

Reply:
See my latest comment for a way to fix that.

Reply:
Thanks, this looks great! I'll do that!